adapter/src/main/java/org/jvalue/ods/adapterservice/datasource/api/rest/v1/Mappings.java

```diff
@@ -0,0 +1,5 @@
+package org.jvalue.ods.adapterservice.datasource.api.rest.v1;
+
+public class Mappings {
+  public static final String DATA_PATH = "/data";
```

Comment:
Maybe add the path `/datasources` from the `DatasourceEndpoint` here to, so all path mappings are defined in one place.

Reply:
good point!